app/views/rating/applications/show.html.slim

```diff
@@ -43,6 +43,8 @@
         td = format_application_flags(@application)
 
   h2 Submitted Application
+  = show_student_names(@application)
```

Comment:
I just unrolled the current structure. it's now way easier to make those kind of changes. I won't include them in this PR to keep it focused on the core issue.